zerver/openapi/openapi.py

```diff
@@ -332,10 +332,11 @@ def get_openapi_parameters(
 def get_openapi_return_values(endpoint: str, method: str) -> Dict[str, Any]:
     operation = openapi_spec.openapi()["paths"][endpoint][method.lower()]
     schema = operation["responses"]["200"]["content"]["application/json"]["schema"]
-    # In cases where we have used oneOf, the schemas only differ in examples
-    # So we can choose any.
-    if "oneOf" in schema:
-        schema = schema["oneOf"][0]
+    # We do not currently have documented endpoints that have multiple schemas
+    # ("oneOf", "anyOf", "allOf") for success ("200") responses. If this changes,
+    # then the assertion below will need to be removed, and this function updated
+    # so that endpoint responses will be rendered as expected.
+    assert "properties" in schema
```

Comment:
Due to the changes to the coverage configuration (#25824), the above check would have needed a `#nocoverage` comment with these changes because the "partially_completed" success responses were the only ones using (incorrectly) "oneOf" in the API documentation.

The comment above is not correct in how "oneOf" should be used with response schemas, so I thought it better to add an assertion here for now. And if we do add documentation for endpoints that have varying success response schemas, then we can update this function as noted in the comment.

FYI - If a schema only differs in examples, then we should be using [the example object](https://spec.openapis.org/oas/v3.0.3#example-object), as noted in 02c1196e3e79ef8a4f29638ceb8c09e8335c550f (with the same reason as to why we currently do not use them).

Reply:
OK, that works for me.